process/check.go

```diff
@@ -0,0 +1,41 @@
+package process
+
+import elasticIndexer "github.com/ElrondNetwork/elastic-indexer-go"
+
+func checkArguments(arguments *ArgElasticProcessor) error {
+	if arguments == nil {
```

Comment:
this could have been avoided if the arguments would have been of type `ArgElasticProcessor` not `*ArgElasticProcessor`